### DISCUSSION TITLE
Held note tail on upscroll not appearing

### DISCUSSION THREAD

**Discussion by animrei** [ppy/osu#22778] [Q&A] [answered]
So basically, the held note tail on upscroll (The whole stretchy thing on the held note basically) and makes it look like a normal note. This issue however, does not occur on downscroll on my current skin. This issue is not on the default skin (argon) at all so it is might be osu!lazer having lack of support for certain skins?

**Comment by Theighlin** [comment]
Fixed by #22776 if it gets merged. Heard there's gonna be a hotfix release soon.

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/22768
